lib/inspec/backend.rb

```diff
@@ -1,9 +1,4 @@
 # encoding: utf-8
-# copyright: 2015, Dominik Richter
```

Comment:
I think you cannot remove the copyright, since it stays available. This was developed by Dom before it was acquired by Chef